pull_request_template.md

```diff
@@ -1,21 +1,32 @@
-Closes: [#XX](url)
+# Pull Request Name: [#197](https://github.com/Fondeadora/fondeadora/issues/197)
 
-*_RFC_: *
+**Note**: Use your preferred `keyword`, but jus one (Don't forget to delete the `<br/>` and the rest of options).
 
-**Descripción**
-Un buen lugar para discutir implementación, diseño, etc. 
-Entre más complejo más vale la pena abrirlo temprano (facilita la revisión)
+Close Fondeadora/fondeadora#197 <br/>
+Closes Fondeadora/fondeadora#197 <br/>
+Closed Fondeadora/fondeadora#197 <br/>
+Fix Fondeadora/fondeadora#197 <br/>
+Fixes Fondeadora/fondeadora#197 <br/>
+Fixed Fondeadora/fondeadora#197 <br/>
+Resolve Fondeadora/fondeadora#197 <br/>
+Resolves Fondeadora/fondeadora#197 <br/>
+Resolved Fondeadora/fondeadora#197 <br/>
 
-**¿Cuál es el comportamiento actual?**
+## Description
```

Comment:
Creo que Ernesto tenía de referencia [este artículo](https://embeddedartistry.com/blog/2017/8/4/a-github-pull-request-template-for-your-projects) en donde están descritas estas secciones más claramente (y ya en inglés). Podemos copiar de ahí las relevantes.

Reply:
Muy bien lo reviso y agrego lo que haga falta